server/src/main/java/org/elasticsearch/common/breaker/ChildMemoryCircuitBreaker.java

```diff
@@ -68,7 +68,7 @@ public ChildMemoryCircuitBreaker(
     public void circuitBreak(String fieldName, long bytesNeeded) {
         final long memoryBytesLimit = this.limitAndOverhead.limit;
         this.trippedCount.incrementAndGet();
-        this.trippedCountMeter.increment();
+        this.trippedCountMeter.add(1L);
```

Comment:
Do we still need to increment the counter by one with up down counters?

Reply:
I was confused by `add(1L)`, thinking that it would always initialise these counters to 1L, but this just increments the counter.

Reply:
`add` accepts a value which might also be negative for an up/down counter.